Makefile

```diff
@@ -314,11 +331,11 @@ install : $(libceed) $(OBJDIR)/ceed.pc
 	$(INSTALL_DATA) $(OBJDIR)/ceed.pc "$(DESTDIR)$(pkgconfigdir)/"
 	$(if $(OCCA_ON),$(INSTALL_DATA) $(OKL_KERNELS) "$(DESTDIR)$(okldir)/")
 
-.PHONY : all cln clean print test tst prove prv examples style install doc okl-cache okl-clear
+.PHONY : cln clean print test tst prove prv examples style install doc okl-cache okl-clear
 
 cln clean :
-	$(RM) *.o *.d $(libceed)
-	$(RM) -r *.dSYM $(OBJDIR) $(LIBDIR)/pkgconfig
+	$(RM) $(libceed)
+	$(RM) -r $(OBJDIR) $(LIBDIR)/pkgconfig
```

Comment:
We can just delete the entire `$(LIBDIR)` and thus won't need to specifically delete `libceed`.

Reply:
We can certainly do that.